week_1/Array/move_zeroes.js

```diff
@@ -0,0 +1,22 @@
+// Given an integer array nums, move all 0's to the end of it while maintaining the relative order of the non-zero elements.
+
+// Note that you must do this in-place without making a copy of the array.
+/**
+ * @param {number[]} nums
+ * @return {void} Do not return anything, modify nums in-place instead.
+ */
+const moveZeroes = (nums) => {
```

Comment:
looks like there is two duplicate files here?

Reply:
I got rid of the one in the Array folder.